src/main/java/com/noroff/mefit/controller/ExerciseController.java

```diff
@@ -0,0 +1,39 @@
+package com.noroff.mefit.controller;
+
+import com.noroff.mefit.model.Exercise;
+import com.noroff.mefit.service.ExerciseService;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@Tag(name = "Exercise")
+@RequestMapping("/api/exercise")
+public record ExerciseController(ExerciseService exerciseService) {
+
+    @GetMapping
+    public List<Exercise> getAllExercises() {
+        return exerciseService.getAllExercises();
+    }
+
+    @PostMapping
+    public Exercise create(@RequestBody Exercise exercise) {
+        return exerciseService.create(exercise);
+    }
```

Comment:
Change due to inconsistency. Not breaking changes.
```suggestion
    @PostMapping
    public Exercise createExercies(@RequestBody Exercise exercise) {
        return exerciseService.create(exercise);
    }
```